java/src/main/java/ai/rapids/cudf/CudaComputeMode.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ai.rapids.cudf;
+
+/**
+ * This is the Java mapping of CUDA device compute modes.
+ */
+public enum CudaComputeMode {
+  cudaComputeModeDefault(0),
```

Comment:
it would be nice to include a comment per enum value to match the documentation, like we do for `CudaMemcpyKind` or `CompressionType`.

Reply:
Documents added.